source/generate_dict/generate_uid_dict.py

```diff
@@ -27,7 +27,10 @@
 from pydicom import _version
 
 
-PYDICOM_DICT_FILENAME = '../../pydicom/_uid_dict.py'
+PYDICOM_DICT_FILENAME = os.path.join(
```

Comment:
This part of the code is different in each of the `source/generate_*.py` scripts. Perhaps use a different commit or PR to homogenize this part of the code?
* https://github.com/pydicom/pydicom/blob/59c23a252643b9ccf0169f2e3d5c5f421e5f946e/source/generate_uids/generate_storage_sopclass_uids.py#L91-L92
* https://github.com/pydicom/pydicom/blob/59c23a252643b9ccf0169f2e3d5c5f421e5f946e/source/generate_dict/generate_dicom_dict.py#L58-L61
* https://github.com/pydicom/pydicom/blob/59c23a252643b9ccf0169f2e3d5c5f421e5f946e/source/generate_dict/generate_private_dict.py#L98-L103
* * https://github.com/pydicom/pydicom/blob/59c23a252643b9ccf0169f2e3d5c5f421e5f946e/source/generate_dict/generate_uid_dict.py#L30

Reply:
Addressed via f0ef21c5366ab2c7a6fa86c0bd2d175120dd2ba2